cli/src/pcluster/validators/tags_validators.py

```diff
@@ -61,3 +63,10 @@ def _validate(self, queue_name, compute_resource_name, cluster_tags, queue_tags,
                 f" '{queue_name}'.",
                 FailureLevel.WARNING,
             )
+
+        if key_count > MAX_TAGS_COUNT:
+            self._add_failure(
+                f"The number of tags ({key_count}) associated with ComputeResource '{compute_resource_name}' in queue "
+                f"'{queue_name}' has exceeded the limit of {MAX_TAGS_COUNT}.",
+                FailureLevel.ERROR,
```

Comment:
Is the tag limit on the queue level or cluster level? My understanding is it's the total number combining all 3 levels, if that's the case, how do we determine which queue to report here?

Reply:
it's the total number combining all 3 levels, we do the check for each queues, the validator is under cluster config

Reply:
The error message here seems to imply the problem comes from the Compute resource level because of the wording "_the number of tags associated with ComputeResource xxx_". While I understand the "_tags associated with ComputeResource xxx_" include the tags from the cluster level, queue level, and compute resource level, so it's technically correct, I do feel there is better way to communicate the information to users. Something like this (note this is just an example)

"_The total number of tags ({key_count}) defined under `Tags`, `{queue_name}/Tags` and `{queue_name}/{compute_resource_name}/Tags` has exceeded the limit of {MAX_TAGS_COUNT}._"

This is just a suggestion, not a blocking comment, so feel free to go with your preferred solution.

Reply:
Thank you for your suggestion, however, {queue_name}/{compute_resource_name} is not a section name, usually we just write the section name, please see examples in other verification messages, I have indicated which queue or which computing resources in the verification message

Reply:
That was just an example to express the idea. The point was to make clear the total number comes from all 3 levels. Is it better now:

"_The total number of tags ({key_count}) defined under `Tags`, `SlurmQueue/Tags` and `SlurmQueue/ComputeResources/Tags` has exceeded the limit of {MAX_TAGS_COUNT} for ComputeResource {compute_resource_name} in queue {queue_name}._" ?

Reply:
The message you suggested can cause misunderstanding, it is possible they don't have queue tags, or they don't have compute resource tags, or they have tags under other queues.

Thank you for your suggestions, I believe that the level of detail provided in the current validation message is sufficient to achieve the desired outcome of preventing cluster creation in case of an error in this corner case. I will go with my preferred solution.